zktest/src/archive/test2/F85-ZK-1148.zul

```diff
@@ -22,4 +22,10 @@
 	<button id="delBtno" label="detach out window" onClick='outWin.detach();'/>
 	<button label="attach window" onClick=''/>
 	<label id="resultLabel"></label>
+	<separator/>
+	<window id="reloadWin"
+		viewModel="@id('vm')@init('org.zkoss.zktest.test2.F85_ZK_1148ReloadVM')">
+		<include id="inc1" src="@load(vm.url) @converter(vm.converter)"/>
+		<button id="reload" label="reload" onClick="@command('reload')"/>
```

Comment:
You added a new button, but no instruction provided. No one else knows how to test and what's the expected result.